cumulus_library/studies/core/builder_condition.py

```diff
@@ -54,6 +54,12 @@ def denormalize_codes(self):
                 "http://snomed.info/sct",
                 "http://hl7.org/fhir/sid/icd-10-cm",
                 "http://hl7.org/fhir/sid/icd-9-cm",
+                "http://hl7.org/fhir/sid/icd-9-cm/diagnosis",
+                # Cerner specific systems
```

Comment:
That doesn't seem entirely true - it's a spec standard way of indicating why data is gone. Epic doesn't use it though, it sounds like.

Reply:
oh good catch